src/maximize.ts

```diff
@@ -18,6 +18,7 @@ import {
 import { $familiar, $item, $slot, $slots, $stats } from "./template-string";
 import logger from "./logger";
 import { setEqual } from "./utils";
+import merge from "lodash/merge";
```

Comment:
I just merged a `import { maxBy } from "lodash"`. Which one is the correct import style for minimizing bundle size and can we make a change one way or the other to reflect.

Also put this import at the top since its from an external module

Reply:
If I had to guess, probably importing from lodash/merge and lodash/maxBy would minimize bundle size. Absolutely will move it to the top right now.